### PR TITLE
Fix Docker admin asset build

### PR DESCRIPTION
**Pull request by amalshaji**
## Summary
- separate the admin TypeScript/Vite bundle from the repository-wide prebuild asset check
- use the bundle-only command inside the isolated Docker web-builder stage
- add a pull-request Docker web-builder smoke test so this filesystem boundary is checked before merge

## Root cause
The admin prebuild hook referenced ../../../../scripts/sync-brand-assets.sh, but the Docker stage copies only the admin package into /app/web. The relative path therefore resolved to a script that was not present in the image.

## Validation
- ./scripts/sync-brand-assets.sh --check
- bun run lint (0 errors; 7 existing warnings)
- bun run test (13 tests passed)
- bun run build
- docker build --target web-builder .
- actionlint -ignore SC2046 .github/workflows/portr-server.yml
- make buildgo
- make testgo

Failed run: https://github.com/amalshaji/portr/actions/runs/30839738780/job/91773571227

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -8,7 +8,7 @@ RUN bun install --frozen-lockfile
 
 COPY internal/server/admin/web-v2 ./
 
-RUN bun run build
+RUN bun run build:app
```
[the remark this answers is not in the document]
Comment (amalshaji) [reply Dockerfile:12]: Fixed in 52d77e7. The Docker stage now overwrites the package copy with assets/brand/portr-mark.svg before bundling. Verified with docker build --target web-builder and an in-container cmp between the public and emitted static assets.
